main.qml

```diff
@@ -1989,7 +1989,9 @@ ApplicationWindow {
     }
 
     function checkUpdates() {
-        walletManager.checkUpdatesAsync("monero-gui", "gui")
+        if (!disableCheckUpdatesFlag) {
```

Comment:
I could also additionally disable `updatesTimer`.

Reply:
I would omit this change, drop `checkUpdates` from `onCompleted` and use `triggeredOnStart` and `running` `updatesTimer`'s properties instead

Reply:
Updated.